### DISCUSSION TITLE
Default Icon for files in root directory?

### DISCUSSION THREAD

**Discussion by HiBecca** [FlorianWoelki/obsidian-icon-folder#191] [Q&A] [answered]
Is it possible to set an inherited icon for files that aren't in a folder?

**Comment by FlorianWoelki** [comment, marked as answer]
With the functionality of the inherited icon, it is not possible. However, I think this is doable through something like Custom Rules. Feel free to check them out, you can insert different regex.

**Reply by HiBecca** [reply]
Oh sweet, I googled it and `(.*?)` 'matches everything', I set the rule to 'apply to only files' and it worked like a charm, thanks!

**Reply by Niehaus** [reply]
Worked really well. I set two rules as `(.*?)` with different icons, one for only files and other for only folders, worked nicely!

**Reply by AndreM222** [reply]
But what did you use to get the folders only?

**Reply by Niehaus** [reply]
Only add the rule and change that third icon (right-left reading) and then you can choose for which type of file/folder they will apply!
![image](https://github.com/FlorianWoelki/obsidian-icon-folder/assets/17555995/6f7b5a9d-e7f6-436f-b7a7-e45a2998ee48)

**Reply by AndreM222** [reply]
thanks. did not notice I could change the option with the right botton
